docs/source/datasets/index.rst

```diff
@@ -461,4 +455,148 @@ The following columns are included in these datasets:
+  Add link to docstring once ready.
+
+The following columns are included in these datasets:
+
+.. list-table:: **Dataset columns**
+   :header-rows: 1
+
+   * - Attribute Name
+     - Column Name
+     - Notes
+   * - Unique simulant ID
+     - :code:`simulant_id`
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation.
+   * - Unique household ID
+     - :code:`household_id`
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation; consistent across all
+       datasets. 
+   * - First name
+     - :code:`first_name`
+     - 
+   * - Middle initial
+     - :code:`middle_initial`
+     - 
+   * - Last name
+     - :code:`last_name`
+     - 
+   * - Mailing address street number
+     - :code:`mailing_address_street_number`
+     - 
+   * - Mailing address street name
+     - :code:`mailing_address_street_name`
+     - 
+   * - Mailing address unit number
+     - :code:`mailing_address_unit_number`
+     - 
+   * - Mailing address PO box
+     - :code:`mailing_address_po_box`
+     - 
+   * - Mailing address city
+     - :code:`mailing_address_city`    
+     - 
+   * - Mailing address state
+     - :code:`mailing_address_state`  
+     - 
+   * - Mailing address ZIP code
+     - :code:`mailing_address_zipcode`
+     - 
+   * - Social security number 
+     - :code:`ssn`
+     - Individual Taxpayer Identification Number (ITIN) if no SSN
+   * - Income 
+     - :code:`income`
+     - 
+   * - Employer ID 
+     - :code:`employer_id`
+     -  
+   * - Employer Name 
+     - :code:`employer_name`
+     - 
+   * - Employer street number
+     - :code:`employer_street_number`
+     - 
+   * - Employer street name
+     - :code:`employer_street_name`
+     - 
+   * - Employer unit number
+     - :code:`employer_unit_number`
+     - 
+   * - Employer city
+     - :code:`employer_city`    
+     - 
+   * - Employer state
+     - :code:`employer_state`  
+     - 
+   * - Employer ZIP code
+     - :code:`employer_zipcode`
+     - 
+   * - Joint filer first name
+     - :code:`spouse_first_name`
+     - 
+   * - Joint filer middle initial
+     - :code:`spouse_middle_initial`
+     - 
+   * - Joint filer last name
+     - :code:`spouse_last_name`
+     - 
+   * - Joint filer social security number
+     - :code:`spouse_ssn`
+     - Individual Taxpayer Identification Number (ITIN) if no SSN
+   * - Joint filer income
+     - :code:`spouse_income`
+     - 
+   * - Joint filer employer ID
+     - :code:`spouse_employer_id`
+     - 
+   * - Joint filer employer name
+     - :code:`spouse_employer_name`
+     - 
+   * - Joint filer employer address
+     - :code:`spouse_employer_address`
+     - 
+   * - Joint filer employer ZIP code
+     - :code:`spouse_employer_zipcode`
+     - 
+   * - Dependent 1 first name
+     - :code:`dependent_1_first_name`
+     - 
+   * - Dependent 1 last name
+     - :code:`dependent_1_last_name`
+     - 
+   * - Dependent 1 social security number
```

Comment:
```suggestion
   * - Dependent 1 Social Security Number (SSN)
```

You could do this, **or** use just "SSN" since you will have introduced that acronym above. I kind of like spelling it out again, but I don't feel strongly.